### PR TITLE
applications: nrf5340_audio: Add manufacturer ID to adv

### PR DESCRIPTION
**Pull request by alexsven**
- Add manufacturer ID UUID to the advertisement packet
- OCT-2730

### CODE REVIEW

File: applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_source.c [hunk 1]
```diff
@@ -202,9 +202,17 @@ static int adv_create(void)
 	NET_BUF_SIMPLE_DEFINE_STATIC(brdcst_id_buf, BT_UUID_SIZE_16 + BT_AUDIO_BROADCAST_ID_SIZE);
 	/* Buffer for Appearance */
 	NET_BUF_SIMPLE_DEFINE_STATIC(brdcst_appearance_buf, BT_UUID_SIZE_16);
+	/* Buffer for manufacturer ID */
+	NET_BUF_SIMPLE_DEFINE_STATIC(manufacturer_id_buf, BT_UUID_SIZE_16);
 	/* Buffer for Public Broadcast Announcement */
 	NET_BUF_SIMPLE_DEFINE_STATIC(base_buf, 128);
 
+	net_buf_simple_add_le16(&manufacturer_id_buf, CONFIG_BT_DEVICE_MANUFACTURER_ID);
```
Comment (koffes) [inline applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_source.c:210]: I don't quite see why this is added directly here, whilst there is a function created to do the same thing in another location.

File: applications/nrf5340_audio/src/bluetooth/bt_management/bt_mgmt.h [hunk 2]
```diff
@@ -80,6 +80,17 @@ enum bt_mgmt_scan_type {
...
+
+/**
+ * @brief	Add manufacturer ID UUID to the advertisement packet.
+ *
+ * @param[out]	uuid_buf	Buffer being populated with UUIDs.
+ * @param[in]	company_id	16 bit UUID specific to the company.
+ *
+ * @return	0 for success, error otherwise.
+ */
+int bt_mgmt_manufacturer_uuid_populate(struct net_buf_simple *uuid_buf, uint16_t company_id);
```
Comment (koffes) [inline applications/nrf5340_audio/src/bluetooth/bt_management/bt_mgmt.h:92]: Do we need to add this function?

**Review by koffes — APPROVED** [review]

File: applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_source.c (hunk 1 above)
Comment (alexsven) [reply applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_source.c:210]: Created OCT-2949 to get these more similar

File: applications/nrf5340_audio/src/bluetooth/bt_management/bt_mgmt.h (hunk 2 above)
Comment (alexsven) [reply applications/nrf5340_audio/src/bluetooth/bt_management/bt_mgmt.h:92]: I think it's cleaner

**Review by rick1082 — APPROVED** [review]

**Pull request merged**